lib/hooks/useGCFormContext.tsx

```diff
@@ -139,7 +148,7 @@ export const GCFormsProvider = ({
     return values.current as FormValues;
   };
 
-  // TODO: once groups flag is on, just use formHasGroups
+  // TODO: once groups flag is on, just use formHasGroups - tracked in #4407
   const groupsCheck = (groupsFlag: boolean | undefined) => {
```

Comment:
Noting the flag will be always on however a legacy form will not have a groups prop or a groups layout etc...